model/layout.md

```diff
@@ -47,21 +50,47 @@ Determines if the layer has fixed or automatic length for its axis which is orth
 
 **Kind**: instance property of [<code>Layout</code>](#Layout)
 
-<a name="Layout+alignment"></a>
-### layout.alignment : <code>String</code>
+<a name="Layout+itemAlignment"></a>
+### layout.itemAlignment : <code>String</code>
 Determines how the children are aligned inside the layer.
 
 When layout direction is `"row"`:
 - `"min"`: Align the children to the top of the layer
 - `"max"`: Align the children to the bottom of the layer 
 - `"stretch"`: Stretch the children to fill the height of the layer 
 - `"center"`: Vertically center the children
+- `"inherit"`: 
+- `"baseline"`: 
 
 When layout direction is `"column"`:
 - `"min"`: Align the children to the left of the layer
 - `"max"`: Align the children to the right of the layer 
 - `"stretch"`: Stretch the children to fill the width of the layer 
 - `"center"`: Horizontally center the children
+- `"inherit"`: 
+- `"baseline"`: 
```

Comment:
@SamilCan can you help with explanations for `inherit` and `baseline` for both `row` and `columns` cases?

Reply:
You can find layout alignment [doc]( https://www.figma.com/plugin-docs/api/properties/nodes-layoutalign/#signature  ) and itemAlignment [doc](https://www.figma.com/plugin-docs/api/properties/nodes-counteraxisalignitems/#signature). We are using one Alignment Enum to define both of these props because they share most of their values.